assets/ports_lists.json

```diff
@@ -8998,7 +8998,7 @@
             "tcp": true
         },
         {
-            "description": "The Palace \"The Palace\" Virtual Reality Chat software.\u00e2\u20ac\u201d5",
+            "description": "The Palace \"The Palace\" Virtual Reality Chat software. 5",
```

Comment:
Would like to know what does the 5 mean, it is after the "." and I am not sure if it is part of the description of the protocol